scripts/simulator_base.js

```diff
@@ -703,7 +703,8 @@ var SIMULATOR = {};
 			}
 
 			// No Targets
-			if (!targets.length) return 0;
+			// Bolt always resets counter, even if it had no targets
+			if (!targets.length) return true;
```

Comment:
I should have also posted here, when doing this test in the game with 0 targets it actually did not reset the counter (and tried again next turn).

Reply:
Addressed in #127